mongo/client_bulk_write_models.go

```diff
@@ -105,6 +106,14 @@ func (uom *ClientUpdateOneModel) SetUpsert(upsert bool) *ClientUpdateOneModel {
 	return uom
 }
 
+// SetSort specifies which document the operation updates if the query matches multiple documents. The first document
+// matched by the sort order will be updated. This option is only valid for MongoDB versions >= 8.0. The driver will
+// return an error if the sort parameter is a multi-key map. The default value is nil.
```

Comment:
Suggest clarifying this sentence:

> The driver will return an error if the sort parameter is a multi-key map.

Note that the sort parameter cannot be an unordered since different orders mean different things. Specifically, when sorting on multiple fields, sort order is evaluated from left to right.

>  Because sort order matters (evaluating left to right), the driver will return an error if the sort parameter is a multi-key map to prevent unintentional behavior.

SetSort for ClientReplaceOneModel also needs to be updated.

Reply:
I will update the comment for `findoptions`, too.